ext/http/lib.rs

```diff
@@ -1687,3 +1689,63 @@ fn extract_network_stream<U: CanDowncastUpgrade>(
   drop(upgraded);
   unreachable!("unexpected stream type");
 }
+
+#[op2]
+#[serde]
+pub fn op_http_serve_address_override(
+) -> (Option<String>, Option<String>, Option<u16>) {
+  match std::env::var("DENO_SERVE_ADDRESS") {
+    Ok(val) => parse_serve_address(&val),
+    Err(_) => (None, None, None),
+  }
+}
+
+fn parse_serve_address(
```

Comment:
This magic parsing doesn't work because it can't handle `vsock`. We should require that these are prefixed:

- `tcp:<addr>:<port>`
- `unix:<path>`
- `vsock:<cid>:<port>`

Reply:
https://github.com/denoland/deno/pull/28941